scripts/packages/e2e-test-create-block-app.ts

```diff
@@ -136,7 +136,7 @@ const script = async () => {
       },
     });
 
-    await waitOn({ resources: ["http://localhost:63212"], timeout: 20000 });
+    await waitOn({ resources: ["http://localhost:63212"], timeout: 30000 });
```

Comment:
Helps pass CI on slow GitHub machines (which we may be assigned at random).

- [Successful run](https://github.com/blockprotocol/blockprotocol/runs/7974162736?check_suite_focus=true#step:7:169) (≈ 15s)
- [Failed run](https://github.com/blockprotocol/blockprotocol/runs/7976345609?check_suite_focus=true#step:7:189) (hitting 20s timeout)

Looking at task durations in the above runs shows that the second machine is generally slower.